### PR TITLE
basic setup for membership page

### LINKED ISSUE #3: Add 'Membership' subpage
## Add 'Membership' subpage

Add the *'Membership'* subpage that contains crucial information about the structure fo `SKIP`, board members, regular members and **how** `SKIP` works as an organisation. Make use of the provided **diagram**; the text is given in the shared document.

### PR DESCRIPTION
**Pull request by JonathanPartain**
Feedback and/or improvements welcome. Aims to solve #3

### CODE REVIEW

File: membership.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+## SKIP Student Association Structure Overview
+
+The SKIP Student Association operates in collaboration with Göta Studentkår and is designed to enhance the student experience by organizing events. To effectively achieve its mission, SKIP comprises two distinct categories of members: "Board Members" and "Regular Members." It's important to note that all members, both board and regular, must also be members of Göta Studentkår.
+
+![Skip structure](/assets/imgs/skip-structure.png)
+
+
+[comment]: <> (Be wary of whitespace after list items)
+1.
```
Comment (michalspano) [inline membership.md]: I'd suggest to include the **membership type** after the ordered list items, just like in the template!

File: membership.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+title: Membership
+layout: post
+---
+
+## SKIP Student Association Structure Overview
+
+The SKIP Student Association operates in collaboration with Göta Studentkår and is designed to enhance the student experience by organizing events. To effectively achieve its mission, SKIP comprises two distinct categories of members: "Board Members" and "Regular Members." It's important to note that all members, both board and regular, must also be members of Göta Studentkår.
+
+![Skip structure](/assets/imgs/skip-structure.png)
```
Comment (michalspano) [inline membership.md:10]: Good! I'm just wondering that it doesn't look too well under the white background, perhaps I'll wrap the diagram in a rectangle with some border shadow, or what not!

Comment (michalspano) [reply membership.md:10]: Or, alternatively, we could change the theme; the darker themes look more appealing anyway!

**Review by michalspano — COMMENTED** [review]
Left a few comments!

File: membership.md (hunk 2 above)
Comment (JonathanPartain) [reply membership.md:10]: Thats up to you! But I do agree that the dark theme looks better :)

Comment (JonathanPartain) [reply membership.md:10]: I tried out the different themes, and night definitely suited best imo.

Comment (michalspano) [reply membership.md:10]: On it!

Comment (JonathanPartain) [reply membership.md:10]: I added the file in this PR, so if it gets deployed it should be all good

Comment (michalspano) [reply membership.md:10]: I do agree! Looks great.

**Review by michalspano — APPROVED** [review]
All appears well now – **approved**!

**Pull request merged**